pkg/security/secl/model/category.go

```diff
@@ -38,7 +38,7 @@ func GetAllCategories() []EventCategory {
 // GetEventTypeCategory returns the category for the given event type
 func GetEventTypeCategory(eventType eval.EventType) EventCategory {
 	switch eventType {
-	case "exec", "signal", "exit", "fork", "anomaly_detection_syscall":
+	case "exec", "signal", "exit", "fork", "anomaly_detection_syscall", "syscalls":
```

Comment:
Do we still need `"anomaly_detection_syscall"` ?

Reply:
Nop, good catch thanks !